modules/op-release-notes-1-8.adoc

```diff
@@ -20,6 +20,44 @@ With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift
 
 include::snippets/technology-preview.adoc[]
 
+=== Adding Git repositories by using webhooks
+
+With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible.
+// (link:https://issues.redhat.com//browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+=== Running multiple PipelineRun objects
```

Comment:
Suggestion: "Running multiple PipelineRuns on a Git event"

"Previously, as a developer, you could run only one `PipelineRun` from your `.tekton` directory for a single Git event e.g. pull-request, ..."

Reply:
Updated. Thanks.